tools/backup/backup.sh

```diff
@@ -74,6 +74,19 @@ function cloud_copy() {
     gsutil cp $backup_path $bucket_path
     gsutil cp $backup_path "${bucket_path}${LATEST_POINTER}"
     ;;
+  azure)
+    az storage container create -n "$BUCKET-$database" --account-name $ACCOUNT_NAME --subscription $SUBSCRIPTION
+    az storage blob upload --container-name "$BUCKET-$database" \
+                       --file "$backup_path" \
+                       --name $(basename "$backup_path") \
+                       --account-name $ACCOUNT_NAME \
```

Comment:
throughout this PR, you should double quote whenever you use an env var, to account for the possibility that an env var's value could contain spaces, and break the interpretation of what the bash command means.

I know this *usually* won't be the case (particularly for things like subscription IDs) but it does sometimes come up, and will cause the backup script to break in hard to find ways.    If for example "ACCOUNT_NAME" was "hello world" it'll just give you a syntax error on az.

Reply:
This has been fixed now.